frontend/src/Routes/DynamicRoute/DynamicRoute.tsx

```diff
@@ -57,21 +43,26 @@ const DynamicRoute: React.FC<DynamicRouteProps> = ({ location }) => {
               };
             }
           }),
-        );
-      }
-    }
-  }, [location, dynamicRoutePages]);
+        };
+      }),
+    [dynamicRoutePages],
+  );
 
   return (
-    <section className={currClassName || ''}>
-      {Component ? (
-        <React.Suspense fallback={null}>
-          <Component />
-        </React.Suspense>
-      ) : (
-        <Loader />
-      )}
-    </section>
+    <React.Suspense fallback={null}>
+      {routes.map(({ className, Component, uid, ...currCoute }) => (
+        <React.Fragment key={uid}>
+          <Route
```

Comment:
The fragment is unnecessary here. You can put the `key` on the `Route` instead.

Reply:
Ah, forgot to add it from previous request.